stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -15,7 +15,7 @@ function recursive_dotcalls!(ex, args, i=1)
     if !(ex isa Expr) || ((ex.head !== :. || !(ex.args[2] isa Expr)) &&
                           (ex.head !== :call || string(ex.args[1])[1] != '.'))
         newarg = Symbol('x', i)
-        if ex.head === :...
+        if ex isa Expr && ex.head === :...
```

Comment:
```suggestion
        Meta.isexpr(ex, :...)
```

Reply:
I didn't know about this syntax, thanks!

Reply:
There is a missing `if`, let me fix this